### PR TITLE
Update "node-sass" dep from 4.13.1 to 4.14.1

### PR DESCRIPTION
**Pull request by mvorisek**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update "node-sass" dep in lock files to fix assets build in the latest Alpine. Also add npm config check like for PHP in https://github.com/PrestaShop/PrestaShop/pull/23931
| Type?             | improvement
| Category?         | FO
| BC breaks?        | no
| Deprecations?     |no
| Fixed ticket?     | Fixes #23984
| How to test?      | GH CI must pass
| Possible impacts? | no

How this PR was created:

for all npm directories `npm update node-sass` and `npm install` was run

```
./tools/assets/build.sh
>>> Building admin default theme...
/builds/mvorisek/prestashop_eshop/admin-dev/themes/default /builds/mvorisek/prestashop_eshop
> node-sass@4.13.1 install /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/node-sass
> node scripts/install.js
Downloading binary from https://github.com/sass/node-sass/releases/download/v4.13.1/linux_musl-x64-83_binding.node
Cannot download "https://github.com/sass/node-sass/releases/download/v4.13.1/linux_musl-x64-83_binding.node": 
HTTP error 404 Not Found
Hint: If github.com is not accessible in your location
      try setting a proxy via HTTP_PROXY, e.g. 
      export HTTP_PROXY=http://example.com:1234
or configure npm proxy via
      npm config set proxy http://example.com:8080
> node-sass@4.13.1 postinstall /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/node-sass
> node scripts/build.js
Building: /usr/bin/node /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/node-gyp/bin/node-gyp.js rebuild --verbose --libsass_ext= --libsass_cflags= --libsass_ldflags= --libsass_library=
gyp info it worked if it ends with ok
gyp verb cli [
gyp verb cli   '/usr/bin/node',
gyp verb cli   '/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/node-gyp/bin/node-gyp.js',
gyp verb cli   'rebuild',
gyp verb cli   '--verbose',
gyp verb cli   '--libsass_ext=',
gyp verb cli   '--libsass_cflags=',
gyp verb cli   '--libsass_ldflags=',
gyp verb cli   '--libsass_library='
gyp verb cli ]
gyp info using node-gyp@3.8.0
gyp info using node@14.15.4 | linux | x64
gyp verb command rebuild []
gyp verb command clean []
gyp verb clean removing "build" directory
gyp verb command configure []
gyp verb check python checking for Python executable "python2" in the PATH
gyp verb `which` failed Error: not found: python2
gyp verb `which` failed     at getNotFoundError (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:89:16
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:183:21)
gyp verb `which` failed  python2 Error: not found: python2
gyp verb `which` failed     at getNotFoundError (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:89:16
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:183:21) {
gyp verb `which` failed   code: 'ENOENT'
gyp verb `which` failed }
gyp verb check python checking for Python executable "python" in the PATH
gyp verb `which` failed Error: not found: python
gyp verb `which` failed     at getNotFoundError (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:89:16
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:183:21)
gyp verb `which` failed  python Error: not found: python
gyp verb `which` failed     at getNotFoundError (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:13:12)
gyp verb `which` failed     at F (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:68:19)
gyp verb `which` failed     at E (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:80:29)
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:89:16
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/index.js:42:5
gyp verb `which` failed     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/mode.js:8:5
gyp verb `which` failed     at FSReqCallback.oncomplete (fs.js:183:21) {
gyp verb `which` failed   code: 'ENOENT'
gyp verb `which` failed }
gyp ERR! configure error 
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/node-gyp/lib/configure.js:484:19)
gyp ERR! stack     at PythonFinder.<anonymous> (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/node-gyp/lib/configure.js:406:16)
gyp ERR! stack     at F (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:68:16)
gyp ERR! stack     at E (/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:80:29)
gyp ERR! stack     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/which/which.js:89:16
gyp ERR! stack     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqCallback.oncomplete (fs.js:183:21)
gyp ERR! System Linux 4.19.0-16-amd64
gyp ERR! command "/usr/bin/node" "/builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
gyp ERR! cwd /builds/mvorisek/prestashop_eshop/admin-dev/themes/default/node_modules/node-sass
gyp ERR! node -v v14.15.4
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
Build failed with error code: 1
npm WARN default@1.7.5 No description
npm WARN default@1.7.5 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.13.1 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@4.13.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-04-12T12_49_56_351Z-debug.log
make: *** [Makefile:7: assets] Error 1
```

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24014)

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]
LGTM !

**Comment by NeOMakinG** [comment]
Btw, did you try to build assets after updating node-sass ?

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -31,6 +31,13 @@ jobs:
         with:
             node-version: '10.x'
 
+      -   name: Validate npm config
+          run: >-
+            (cd admin-dev/themes/default && npm ci)
+            && (cd admin-dev/themes/new-theme && npm ci)
+            && (cd themes && npm ci)
+            && (cd themes/classic/_dev && npm ci)
```
Comment (PierreRambaud) [inline .github/workflows/lint.yml:39]:
```suggestion
            (pushd admin-dev/themes/default && npm ci && popd)
            && (pushd admin-dev/themes/new-theme && npm ci && popd)
            && (pushd themes && npm ci && popd)
            && (pushd themes/classic/_dev && npm ci && popd)
```

You can't chain cd command like that. it'll not work

Comment (NeOMakinG) [inline .github/workflows/lint.yml:39]: Is this working ⁉️

**Comment by mvorisek** [comment]
> 
> 
> Btw, did you try to build assets after updating node-sass ?

yes, I was able to complete `make install` command and also CI passes :) (isn't it tested there too?)

File: .github/workflows/lint.yml (hunk 1 above)
Comment (mvorisek) [reply .github/workflows/lint.yml:39]: it is, see CI

Comment (mvorisek) [reply .github/workflows/lint.yml:39]: Why 3 commands than 2?

It is working because `()` starts a new sub shell.

Marking as resolved.

**Comment by NeOMakinG** [comment]
> > Btw, did you try to build assets after updating node-sass ?
> 
> yes, I was able to complete `make install` command and also CI passes :) (isn't it tested there too?)

Well, It's tested by the CI as it build assets, but I was asking it naturally ^^

**Comment by mvorisek** [comment]
> Well, It's tested by the CI as it build assets, but I was asking it naturally ^^

Hey, thanks for trusting me more than computers :D

File: .github/workflows/lint.yml (hunk 1 above)
Comment (PierreRambaud) [reply .github/workflows/lint.yml:39]: oh, funny, totally forget this one :scream:

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Reran CI as it crashed because of something else

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @mvorisek

**Pull request merged**